Student/Student/Submissions/SubmissionDetails/SubmissionDetailsPresenter.swift

```diff
@@ -28,18 +28,20 @@ protocol SubmissionDetailsViewProtocol: ColoredNavViewProtocol, SubmissionButton
     var navigationItem: UINavigationItem { get }
 }
 
-class SubmissionDetailsPresenter {
+class SubmissionDetailsPresenter: PageViewLoggerPresenterProtocol {
     let context: Context
     let assignmentID: String
     let userID: String
     let env: AppEnvironment
     weak var view: SubmissionDetailsViewProtocol?
-
     let submissionButtonPresenter: SubmissionButtonPresenter
     var submissionButtonText: String? {
         guard let course = course.first, let assignment = assignment.first else { return nil }
         return submissionButtonPresenter.buttonText(course: course, assignment: assignment, quiz: quizzes?.first, onlineUpload: nil)
     }
+    var pageViewEventName: String {
+        return "/\(context.pathComponent)/\(context.id)/assignments/\(assignmentID)/submissions/\(userID)"
```

Comment:
the `\(context.id)` isn't necessary since the `context.pathComponent` will include it.